### PR TITLE
add new company - Willings Inc

### PR DESCRIPTION
**Pull request by sangdeepganvir**
This is a modified version of, and should adhere to, the [Contributing Guidelines](../../main/.github/CONTRIBUTING.md).

* [ ] This PR contains housekeeping only (URL edits, copy changes etc) - **if this is checked, delete other lines that don't apply**
* [ ] You are an employee of the company mentioned and confirm all included details are correct.
* [x ] You know your alphabet - company is listed in alphabetical order in the README.md file.
* [x ] The company directly hires employees. ** No bootcamps / freelance sites / etc**
* [x ] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
* [x ] A [company profile](/company-profiles/example.md) is included - This is __required__ for new additions.
* [x ] An entry has been added or edited within the [readme.md file](../../main/README.md).
* [x ] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
* [ x] __Region__ details the geographic regions in which this company employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
* [x ] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

**Review by dougaitken — CHANGES_REQUESTED** [review]
Is the company `Willings`, `Willings, Inc`, or `Willings Inc`?

This is important as the company name in the readme needs to match the name of the file and the company name header in the profile file. 

Please update to make these match. If the name is `Willings, Inc` with the comma, mar the file name `willings-inc.md`

**Comment by sangdeepganvir** [comment]
@dougaitken did the required changes. Kindly review it

File: company-profiles/willings-inc.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+Source : https://www.linkedin.com/company/willingsinc/about/
+
+## Remote status
+
+Recruitment/Empowerment support of advanced IT engineers from Indian Institutes of Technology “GALK” [13-ユ-309586]
+
+## Region
+
+India
```
Comment (dougaitken) [inline company-profiles/willings-inc.md:20]: Readme has "Japan" this has India, which is correct @sangdeepganvir ?

**Review by dougaitken — CHANGES_REQUESTED** [review]

File: company-profiles/willings-inc.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## Company size
+
+2-10 employees
+
+Source : https://www.linkedin.com/company/willingsinc/about/
+
+## Remote status
+
+Recruitment/Empowerment support of advanced IT engineers from Indian Institutes of Technology “GALK” [13-ユ-309586]
```
Comment (dougaitken) [inline company-profiles/willings-inc.md:16]: @sangdeepganvir Here is another India reference, can you please fix. Remote status should explain how the company supports remote employees or what they offer for this.

**Review by dougaitken — CHANGES_REQUESTED** [review]

File: company-profiles/willings-inc.md (hunk 2 above)
Comment (sangdeepganvir) [reply company-profiles/willings-inc.md]: @dougaitken please review it.

**Review by dougaitken — APPROVED** [review]

**Pull request merged**